src/controllers/contactController.ts

```diff
@@ -0,0 +1,59 @@
+const ContactModel = require('../models/contact');
```

Comment:
Also, TS uses another kind of import. You should use it too. (See on the link already given 😄)